test/config.js

```diff
@@ -31,7 +31,7 @@ describe('@config@', function () {
   });
 
   it('should install provided @selenium.version@', function (done) {
-    var ver = '^2.53.1';
+    var ver = '2.53.3';
```

Comment:
> mocha --timeout=30000



  @config@
    ✓ should pass confit object as nemo._config (551ms)
    ✓ should install provided @selenium.version@ (5427ms)
    ✓ should throw an error for invalid @invalid.selenium.version@ (1182ms)
    ✓ should export a Configure method
    ✓ should export a Configure method resolving to a Confit object

  @constructor@
    ✓ should return a promise with @noArguments@
    ✓ should return a promise with @noCallback@ (421ms)
    ✓ should throw an error with @noDriverProps@
    ✓ should launch nemo with @noConfigPath@overrideArg@ (8534ms)
    ✓ should launch nemo with @envConfigPath@noOverrideArg@ (6133ms)
    ✓ should launch nemo with @argConfigPath@noOverrideArg@ (8064ms)
    ✓ should launch nemo with @allArgs@ (4094ms)
    ✓ should return the resolved nemo object when the callback is called (505ms)
    ✓ should resolve when a Confit object is the only parameter (562ms)

  @override@
    ✓ @fromEnv@ over config.json data (472ms)
    ✓ @fromArg@ over config.json data (487ms)
    ✓ @builders@ overrides tgtBrowser abstraction (495ms)

  @plugin@
    ✓ should @priorityRegister@ (467ms)
    ✓ should handle @nonexistPlugin@
    ✓ should handle @failedPluginRegistration@ (476ms)

  @proxy@ 
    ✓ should load problem loading page error (469ms)


  21 passing (38s)